test/drivers/helpers/file_opener.cpp

```diff
@@ -3,8 +3,10 @@
 
 file_opener::file_opener(const char* filename, int flags, int dirfd)
 {
+    errno = 0;
     m_fd = syscall(__NR_openat, dirfd, filename, flags, 0);
-	assert_syscall_state(SYSCALL_SUCCESS, "openat", m_fd, NOT_EQUAL, -1);
+    // Using the macro that deals with ENOSYS produces a build issue since GTEST_SKIP returns value in a ctor.
+    _assert_syscall_state(SYSCALL_SUCCESS, "openat", m_fd, NOT_EQUAL, -1);
```

Comment:
not clear who calls `GTEST_SKIP()` here if we use the internal method and not the macro

Reply:
nobody :/ this is just a ctor unfortunately
AFAIK `openat` should be implemented everywhere...hopefully.

Reply:
Anyway, we will notice immediately since it would break the tests.